piConfig.c

```diff
@@ -139,7 +139,8 @@ int process_file(json_parser * parser, struct file *input, int *retlines, int *r
 	while (1) {
 		read = kernel_read(input, buffer + (BUFFLEN - len), len, &input->f_pos);
 		if (read <= 0) {
-			pr_err("kernel_read returned %d: %x, %ld\n", read, (int)input, (long int)input->f_pos);
+			pr_info("process file read(read-ret:%d, f_pos:%ld)\n",
```

Comment:
What is the reason that this should not be an error any more?

Reply:
actually, it will be an error when read < 0, and  reading finished when read = 0

Reply:
The error case is more important IMHO and thus should be printed with ERR priority. However we could use two different messages, one for the error case  and one for the successful completion of read  (as this information might also be helpful if there is an error somewhere later in the process)

Reply:
yes, agree, with two separate messages it is more appropriate.